packages/cli/lab.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { createRoot } from 'react-dom/client';
+import TokenLab from '@terrazzo/token-lab';
+
+const response = await fetch("/api/tokens")
+const tokenFile = await response.text()
+const root = createRoot(document.getElementById('app'));
+
+root.render(<TokenLab tokensFile={tokenFile} onUpdate={async updatedTokens => {
```

Comment:
This is really clever. Love shipping a little wrapper around the lab in the CLI like this. Also with this being a Node.js tool we can use Vite’s default build without too much hassle (I usually find it lacking for packaging npm packages, but here it works great)!